### ISSUE TITLE
dumb-init missing in the image

### ISSUE THREAD

**Issue by johanneskastl** [johanneskastl/renovatebot-leap156-image#1]
```
$ podman run -ti --rm ghcr.io/johanneskastl/renovatebot-leap156-image:main bash     
/usr/local/bin/docker-entrypoint.sh: line 18: exec: dumb-init: not found
$
```

**Comment by johanneskastl** [comment]
fixed in c5d0195 using the statically linked binary from the dumb-init release page.

**Comment by johanneskastl** [comment]
Now using dumb-init built for openSUSE from the `ojkastl_buildservice/opensuse-leap-dumb-init:15.6` container image.
